### PR TITLE
 added email test to validate route data

### PR DESCRIPTION
**Pull request by newcitywave**
added email test to the validate.rout.data.spec

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -85,5 +85,6 @@
   "Previous": "Previous",
   "Next": "Next",
   "I have read and accept the privacy policy. ": "I have read and accept the privacy policy. ",
-  "Required": "(required)"
+  "Required": "(required)",
+  "required": "required"
```
Comment (EricDube) [inline locales/en.json]: Is this part of a sentence or the actual "required" field when an error shows up?

Comment (MarcoGoC) [inline locales/en.json]: these lines should be updated

Comment (newcitywave) [reply locales/en.json]: is it ok to merge?

Comment (newcitywave) [reply locales/en.json]: i am not sure why it is like that my branch does not shoe that

**Pull request merged**